src/test/common/steps/address-lookup-steps.js

```diff
@@ -8,7 +8,7 @@ const { enterPostcodeAndSubmit } = require('./common-steps')
 const { setupPostcodeLookupWithNoResults, setupPostcodeLookupWithResults } = require('../wiremock')
 const { POSTCODE } = require('./constants')
 
-const POSTCODE_WITH_NO_RESULTS = 'BS1 1AA'
+const POSTCODE_WITH_NO_RESULTS = 'BS11AA'
```

Comment:
Previous normalisation PR caused these tests to fail as the postcode sent (BS11AA) no longer matched that of the wiremock stub (BSA 1AA)